### DISCUSSION TITLE
Reconciliation suspend description

### DISCUSSION THREAD

**Discussion by jgrossmac** [fluxcd/flux2#2108] [Proposals]
It would be awesome to make a description field under suspend optional, so we can write up why we are suspending a resource from being reconciled.

**Comment by mihuross** [comment]
The similar functionality can be found in [consul maint](https://www.consul.io/commands/maint) command, i.e. `consul maint -reason "test of new config"`.

**Comment by stefanprodan** [comment]
Use Kubernetes annotations, they are intended to store user-facing things like this one.
